vms/platformvm/state/camino.go

```diff
@@ -47,6 +47,7 @@ var (
 	multisigOwnersPrefix        = []byte("multisigOwners")
 	consortiumMemberNodesPrefix = []byte("consortiumMemberNodes")
 	claimablesPrefix            = []byte("claimables")
+	deferredPrefix              = []byte("deferred")
```

Comment:
imo we should somehow show that this prefix is for other db than other prefixes
may be with comment

Reply:
You mean to somehow show that this prefix is used for `validatorDB` instead of the `baseDB` ?
I agree with the comment. Smth like `// Used for prefixing the validatorsDB`